lib/Header/Header.js

```diff
@@ -616,6 +616,7 @@ module.exports = kind(
 	* @private
 	*/
 	typeChanged: function () {
+		this.addRemoveClass('moon-large-header', this.get('type') == 'large');
```

Comment:
This is probably a micro-optimization, but could we do something like this:
```
if (was) this.removeClass('moon-' + was + '-header');
if (is) this.addClass('moon-' + is + '-header');
```
Not sure if that reduces readability significantly, probably saves us a few comparisons, just a thought now that we have a class for each of the header types.